### ISSUE TITLE
option to set previous messages to display none to prevent repeating

### ISSUE THREAD

**Issue by souporserious** [schne324/live-region#3]
I came across an issue with `announce` having the possibility of getting called multiple times which results in the screen reader repeating previous messages. I tried setting `aria-atomic` to `false`, but then if the same message comes through it won't be announced when I'd like it to be announced.

In another library that implements something similar with live regions [they are setting old messages to display none](https://github.com/paypal/downshift/blob/master/src/set-a11y-status.js#L29) so previous messages won't be announced which seems to work great. Is this something that can be added as an option to this library? I'm guessing we could do something similar and check if the same message comes through and set it to display none. I would love to hear your thoughts 🙂 I can start a PR, just let me know.

**Comment by schne324** [comment]
imo excessive calls to `announce` is not the responsibility of this module and should be addressed on the implementers end. In addition, adding `display: none` to elements within a live region seems like it would yield unpredictable results with screen readers (some may suppress the announcement but some may not).

**Comment by souporserious** [comment]
No worries, sounds good! I think I can achieve the same results by destroying the `LiveRegion` and recreating it which sounds better anyways since they for sure won't get re-announced 😄. Thanks for the quick reply!

**Comment by schne324** [comment]
No problem, I just tagged 1.1.3 with your destroy feature!

**Comment by souporserious** [comment]
Yay 🎉 thank you!!! I really appreciate the quick help and release 🙏.
